test/karma/test-app/delegates-focus/karma.spec.ts

```diff
@@ -1,12 +1,6 @@
 import { setupDomTests, waitForChanges } from '../util';
 
 describe('delegates-focus', function () {
-  if (navigator.userAgent.indexOf('Chrome') === -1 || navigator.userAgent.indexOf('Edge') > -1) {
```

Comment:
this is the only instance of `navigator.userAgent.indexOf` I see in the test code. For firefox (which I re-enabled sometime in the last year), this would always early return (because that left hand side of the conditional would always be true). In the new infra, Jasmine says "Hey, I found a describe with no children, that's a problem!" when this code is present/running on Firefox, hence its removal

`delegatesFocus` _is_ supported by Edge these days https://caniuse.com/?search=delegatesfocus